etc/notes/CHANGES_5.0.0.md

```diff
@@ -16,6 +16,53 @@ The following is a detailed collection of the changes in the major v5 release of
 
 ## Changes
 
+### Dot Notation Typescript Support Removed By Default
+
+**NOTE** This is a **Typescript compile-time only** change. Dot notation in filters sent to MongoDB still work the same. 
+
+Version 4.3.0 introduced Typescript support for dot notation in filter predicates.  For example:
+
+```typescript
+interface Schema {
+  user: {
+    name: string
+  }
+}
+
+declare const collection: Collection<Schema>;
+// compiles pre-v4.3.0, fails in v4.3.0+
+collection.find({ 'user.name': 4 })
+```
+
+This change caused many problems for users, including slow compilation times and compile errors for
+valid dot notation queries.
+
+Driver 5.0 removes type checking on filter predicates.  The preceding example will compile with
```

Comment:
Not all type checking, right, just dot notation type checking?

Reply:
Yeah, just dot notation support